WISE-Community/WISE
# Issue 1741 teacher notebook

Things to test
- Run this db update ```ALTER TABLE notebookItems MODIFY periodId bigint null;```
- Make sure the Teacher Notebook shows up in the Classroom Monitor. You will need to add the "teacherNotebook" field to the project.json. You can find a sample "teacherNotebook" in the #1741 issue. The "teacherNotebook" goes in the root of the project.json. We will implement the Teacher Notebook authoring later.
- Make sure the Teacher Notebook saves and reloads when the teacher uses it in the Classroom Monitor.
- Make sure the Notebook authoring for the Student Notebook still works.
- Make sure the Student Notebook still saves and reloads.
- Make sure the Student Notes and Reports still show up in the Classroom Monitor.

Closes #1741
